appveyor.yml

```diff
@@ -227,6 +227,11 @@ environment:
         TESTING: ON
         DISABLED_TESTS: "!19 !504 !704 !705 ~1056 !1233"
         CONFIG_ARGS: "--enable-debug --enable-werror --enable-alt-svc --disable-threaded-resolver"
+      - APPVEYOR_BUILD_WORKER_IMAGE: "Visual Studio 2019"
+        BUILD_SYSTEM: autotools
+        TESTING: ON
+        DISABLED_TESTS: "!19 !504 !704 !705 ~1056 !1233"
+        CONFIG_ARGS: "--enable-werror"
```

Comment:
--enable-warnings too maybe, which is omitted in the other builds because it's part of --enable-debug?